server/datastore/mysql/hosts.go

```diff
@@ -1099,6 +1103,7 @@ SELECT
 	COALESCE(hsr.execution_id, '') as execution_id
 FROM
     hosts h
+	JOIN host_display_names hdn ON h.id = hdn.host_id
```

Comment:
Need this because we sort by `display_name`.  This is now the main host list does it, which begs the question of why we don't just use the data in `host_display_names` for the lists, but for now we'll just make this consistent.